app/src/main/java/org/simple/clinic/phone/PhoneNumberMasker.kt

```diff
@@ -0,0 +1,7 @@
+package org.simple.clinic.phone
+
+import io.reactivex.Completable
+
+interface PhoneNumberMasker {
```

Comment:
The naming of this interface indicates that it only masks a number, but it also accepts a `Caller` and it is not clear from the interface name why it would do this.

I think we can improve this by changing the interface to just return a phone number as a string, and have two implementations: a `NonOperationalPhoneNumberMasker` and a `TwilioPhoneNumberMasker`. The `NonOperationalPhoneNumberMasker` can then be injected into the controller and we avoid the need for creating a feature flag at all.

What do you think?

Reply:
Hrm, in that case we won't be able to have an abstract function in the interface. What do you think about `MaskedPhoneCaller`?

Reply:
Sounds good to me.